src/cljs/main/broadfcui/components/autosuggest.cljs

```diff
@@ -37,9 +37,12 @@
    (fn [{:keys [locals props]}]
      (let [{:keys [on-submit]} props
            wrapped-on-submit (fn [e]
-                               (on-submit (.. e -target -value)))
+                               (let [value (.. e -target -value)]
+                                 (when-not (empty? value)
+                                   (on-submit value))))
```

Comment:
In browsers that don't fire the search event (firefox), this stops deleting an active search and hitting return from working. Instead, try reverting this line...